BDArmory/UI/BDArmorySetup.cs

```diff
@@ -3313,7 +3313,17 @@ void WindowSettings(int windowID)
                 {
                     if (BDArmorySettings.RUNWAY_PROJECT != (BDArmorySettings.RUNWAY_PROJECT = GUI.Toggle(SLeftRect(++line), BDArmorySettings.RUNWAY_PROJECT, StringUtils.Localize("#LOC_BDArmory_Settings_RunwayProject"))))//Runway Project
                     {
-                        if (HighLogic.LoadedSceneIsEditor && EditorLogic.fetch.ship is not null) GameEvents.onEditorShipModified.Fire(EditorLogic.fetch.ship);
+                        if (HighLogic.LoadedSceneIsEditor && EditorLogic.fetch.ship is not null) GameEvents.onEditorShipModified.Fire(EditorLogic.fetch.ship); if (oldSpaceHacks != BDArmorySettings.SPACE_HACKS)
```

Comment:
Is ` if (oldSpaceHacks != BDArmorySettings.SPACE_HACKS)` a copy-paste error here?

Also, it probably doesn't make any difference at the moment, but the `onEditorShipModified` event should probably be fired after hacking the actuators in case something we do in the future is affected by it.